app/core/src/main/java/com/fsck/k9/AccountPreferenceSerializer.kt

```diff
@@ -24,7 +25,7 @@ class AccountPreferenceSerializer(
             transportUri = Base64.decode(storage.getString("$accountUuid.transportUri", null))
             description = storage.getString("$accountUuid.description", null)
             alwaysBcc = storage.getString("$accountUuid.alwaysBcc", alwaysBcc)
-            automaticCheckIntervalMinutes = storage.getInt("$accountUuid.automaticCheckIntervalMinutes", -1)
+            automaticCheckIntervalMinutes = storage.getInt("$accountUuid.automaticCheckIntervalMinutes", K9IntervalMinutes.INTERVAL_DEFAULT)
```

Comment:
This changes the current default of "never". I'd prefer we keep the old default.